docs/using-with-react.md

```diff
@@ -108,19 +108,21 @@ Callback arguments:
 Callback - called when the mouse moves over the layers.
 
 Arguments:
-- `info` - the `info` object of the topmost
-- `pickedInfos` - an array of info objects for all visible layers that
-matched the picked coordinate, in top to bottom order.
+- `info` - the [info](#remarks) object for the topmost matched layer
+at the coordinate
+- `pickedInfos` - an array of info objects for all visible and pickable 
+layers, in top to bottom order.
 - `event` - the original MouseEvent object
 
 ##### `onLayerClick` (Function, optional)
 
 Callback - called when clicking on the layer.
 
 Arguments:
-- `info` - the `info` object of the topmost
-- `pickedInfos` - an array of info objects for all visible layers that
```

Comment:
Same here - only `pickable` layers that are also `visible` will be considered. Let's use the exact same words.